### PR TITLE
Switch main branch in soteria

### PR DESCRIPTION
**Pull request by pzygielo**
Hi @arjantijms,

I'm not sure how this will work if I open this PR against [soteria](https://github.com/eclipse-ee4j/soteria/) (as I'm not commiter there).
Hope your approval will have the power to move it forward if you agree.

As a lot of PRs in the project are merged to `main`, the 4.0.0 release is done from `main` as well - perhaps the repository default branch could be switched there? From the community POV I find it a bit confusing with current, stalled, outdated `master` to be the default.

Please consider such change.

### CODE REVIEW

**Comment by netomi** [comment]
the change is ok. The eclipse-ee4j organization contains many Eclipse projects and its not possible to associate  which project is responsible for which repo. You are a project lead for some project, so that why you got the information that you can auto-merge, but its ok get approval first from the right project leads before merging.

**Comment by arjantijms** [comment]
@pzygielo I'm okay with the change, thanks for initiating it!

**Review by arjantijms — APPROVED** [review]

**Review by netomi — APPROVED** [review]

**Pull request merged**
